cni/network/network.go

```diff
@@ -362,12 +386,18 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 		telemetry.SendCNIMetric(&cniMetric, plugin.tb)
 
 		// Add Interfaces to result.
-		defaultCniResult := convertInterfaceInfoToCniResult(ipamAddResult.defaultInterfaceInfo, args.IfName)
+		cniResult := cniTypesCurr.Result{}
+		// Add secondaryInterface info to result if it exists; otherwise add default primary interface info to result
```

Comment:
can it be other way.. add secondary interface to result if default interface does not exist

Reply:
in design, we have both default and secondary interface info in ipamAddResult;